README.md

```diff
@@ -54,12 +65,6 @@ On Ubuntu 14.04:
 sudo apt-get install automake autotools-dev fuse g++ git libcurl4-openssl-dev libfuse-dev libssl-dev libxml2-dev make pkg-config
 ```
 
-On CentOS 7:
```

Comment:
Should we remove all compilation instructions from the README and link to the wiki instead?

Reply:
I guess that depends on the policy you want to follow about it :-)

My understanding is that you had CentOS 7 and Ubuntu 14.04 because neither of them had s3fs packages but it was easy to compile them without doing special tricks (such as compiling and installing fuse, as it's required for CentOS 6)

IMHO, I'd keep the compilation instructions, but only generic, and I'd point to the wiki for specific instructions for each distribution.

I can take care of that on a separate PR if you want.

Reply:
Originally we only had compilation instructions until volunteers like you :) contributed packages.  I am fine either keeping all the compilation instructions or moving them to the wiki.  Ubuntu 14.04 will end its support in six months so this should not be top of mind any more!